app/src/main/java/espertolabs/esperto_ble_watch/BLEService.java

```diff
@@ -36,9 +36,11 @@ public BLEService(){
 
     }
 
+    public String deviceAddress;
+
     private final static String TAG = BLEService.class.getSimpleName();
 
-    private static final long SCAN_PERIOD = 10000;
+    private static final long SCAN_PERIOD = 30000;
```

Comment:
what's the point of increasing the scan period here??

Reply:
I was having some issues with it not detecting the watch, increased time to allow it to scan longer before quitting.